Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+> :warning: **WARNING:** We highly suggest you leave iCloud backups on. Without it, if you lose your phone, you will lose all of your seeds. The iCloud backups are encrypted, so no one but you should be able to acces them.
+
+Under the main menu are options to **add** ("+") and **delete** ("edit") seeds followed by a list of each of your seeds, with each seed identified by an Object identity Block ("OID"). You can click the right arrow on a seed to see more data about it and to export it.
+
+> :warning: **WARNING:** Deleting your seeds through the Settings or edit functions will entirely remove them: they will be gone!
+
+## Adding a Seed
+
+Seeds can be created or imported. Seed Creation is done through the **add** ("+") button, while seed importing is done via either the qr button (which imports via camera or other automated means) or the **add** button (which does so via cut-and-pasting).
+
+### Scanning a Seed via Automated Means
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-scan.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-scan.jpeg" align="right" width=250></a>
+
+The **Scan** (qr code) button on the main menu provides the most automated methods for importing seeds, using your camera, the cut-and-paste clipboard, the file system, or your photo roll. 
+
+To scan a QR code, you can either point your camera at the QR of a seed, import a QR of a seed from your Photos, or choose an image of a QR from your Files. 
+
+To scan text defining a seed, just copy it into your Clipboard and click Paste. 
+
+Note that for these methodologies, **Seed Tool** expects the QR code or the clipboard to contain a [Uniform Resource](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md), a standardized way to encode data in an efficient and self-identifying way. This will usually mean a [`ur-crypto-seed`](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-006-urtypes.md#cryptographic-seed-crypto-seed).
+
+Besides scanning seeds, you can also use this view to scan SSKR shares (See "Importing SSKR Shares") or respond to a `ur:crypto-request` (see "Answering Requests"), as described below.
+
+### Importing a Seed via Cut and Paste
+
+The **add** ("+") button below the main menu gives a number of options for creating seeds in **Seed Tool**, but also lets you input text as hex bytes, `ur:crypto-seed`, Bytewords, or BIP39 Mnemonic Words. In each case, you just type or paste the words, and then click "Done".
+
+The following show examples of the data you might input for each data type:
+
+* **Hex Bytes:** 59F2293A5BCE7D4DE59E71B4207AC5D2
+* **`ur:crypto-seed`:** `ur:crypto-seed/oyadgdhkwzdtfthptokigtvwnnjsqzcxknsktdhpyljeda`
+* **Byte Words:** hawk whiz diet fact help taco kiwi gift view noon jugs quiz crux kiln silk tied omit keno lung jade
```

Comment:
Maybe a link here to what are byte words? Or link it the first place we mention byte words?

Reply:
linked all the formats.